proto/Core.proto

```diff
@@ -30,4 +30,25 @@ message TaskEntity {
 
 message Label {
   string label = 1;
+}
+
+message ModelInquiryResult {
```

Comment:
`Inquiry` is more a human term, which is hard to pronounce.
consider `Request`

Reply:
renamed